scales/definitions/cylinder_mm_d281_h180_1-10L_i1.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "./schemas/scale-config",
+    "scale-config": {
+        "containerForm": "cylinder",
+        "lengthUnit": "mm",
+        "volumeUnit": "l",
+        "diameter": 281,
+        "height": 180,
+        "minVolume": 1,
+        "maxVolume": 10,
+        "output": "./scales/svgs/{0}.svg",
+        "description": "old 10 L pot",
```

Comment:
Can you please use the more generic description text (as in other scales)? You can place the additional information in the `successfullyAppliedTo` field.